app-services/json-logic/README.md

```diff
@@ -0,0 +1,54 @@
+# JSON Logic Examples
+
+This example demonstrates a few different ways to leverage JSON logic to assist in a bit more flexible and advanced way to perform filtering operations without the need to write your own functions. If you have many rules or want an even more feature rich rules engine check out [EMQ X Kuiper](https://github.com/emqx/kuiper) here.
+
+> Note: Only  operators that return true or false are supported. See http://jsonlogic.com/operations.html# for the complete list paying attention to return values. Any operator that returns manipulated data is currently not supported. 
+
+> Tip: Leverage http://jsonlogic.com/play.html to get your rule right before implementing in code. JSON can be a bit tricky to get right in code with all the escaped double quotes.  
```

Comment:
We should add this tip to the App Function SDK documentation for JsonLogic function